src/mixins/filing-template-mixin.ts

```diff
@@ -41,7 +42,7 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
   @Getter(useStore) getCreateRulesStep!: CreateRulesIF
   @Getter(useStore) getCreateResolutionStep!: CreateResolutionIF
   @Getter(useStore) getCreateShareStructureStep!: ShareStructureIF
-  @Getter(useStore) getCurrentDate!: string
+  // @Getter(useStore) getCurrentDate!: string
```

Comment:
Already imported via extended mixin.

In other cases I've just deleted these but now I think I like leaving them commented out to show that I thought about it and decided to not include it in the code. Another developer can uncomment this and see the warning. Thoughts?

Reply:
I don't know, I usually just delete them. I guess both are OK. I don't have a strong opinion on this Sev.